src/EventStore.ClientAPI/IEventStoreConnection.cs

```diff
@@ -176,43 +180,44 @@ public interface IEventStoreConnection : IDisposable
         Task<EventReadResult> ReadEventAsync(string stream, long eventNumber, bool resolveLinkTos, UserCredentials userCredentials = null);
 
         /// <summary>
-        /// Reads count Events from an Event Stream forwards (e.g. oldest to newest) starting from position start
+        /// Asynchronously reads count events from an event stream forwards (e.g. oldest to newest) starting from position start.
         /// </summary>
-        /// <param name="stream">The stream to read from</param>
-        /// <param name="start">The starting point to read from</param>
-        /// <param name="count">The count of items to read</param>
-        /// <param name="resolveLinkTos">Whether to resolve LinkTo events automatically</param>
+        /// <param name="stream">The stream to read from.</param>
+        /// <param name="start">The starting point to read from.</param>
+        /// <param name="count">The count of items to read.</param>
+        /// <param name="resolveLinkTos">Whether to resolve LinkTo events automatically.</param>
         /// <param name="userCredentials">The optional user credentials to perform operation with.</param>
         /// <returns>A <see cref="Task&lt;StreamEventsSlice&gt;"/> containing the results of the read operation.</returns>
         Task<StreamEventsSlice> ReadStreamEventsForwardAsync(string stream, long start, int count, bool resolveLinkTos, UserCredentials userCredentials = null);
 
         /// <summary>
-        /// Reads count events from an Event Stream backwards (e.g. newest to oldest) from position asynchronously
+        /// Asynchronously reads count events from an event stream backwards (e.g. newest to oldest) from position.
         /// </summary>
-        /// <param name="stream">The Event Stream to read from</param>
-        /// <param name="start">The position to start reading from</param>
-        /// <param name="count">The count to read from the position</param>
-        /// <param name="resolveLinkTos">Whether to resolve LinkTo events automatically</param>
+        /// <param name="stream">The stream to read from.</param>
+        /// <param name="start">The position to start reading from.</param>
+        /// <param name="count">The count to read from the position.</param>
+        /// <param name="resolveLinkTos">Whether to resolve LinkTo events automatically.</param>
         /// <param name="userCredentials">The optional user credentials to perform operation with.</param>
         /// <returns>A <see cref="Task&lt;StreamEventsSlice&gt;"/> containing the results of the read operation.</returns>
         Task<StreamEventsSlice> ReadStreamEventsBackwardAsync(string stream, long start, int count, bool resolveLinkTos, UserCredentials userCredentials = null);
 
         /// <summary>
-        /// Reads All Events in the node forward asynchronously (e.g. beginning to end)
+        /// TODO: Have we defined node before as a typical word?
```

Comment:
`node` should be common enough for anyone who has got this far to understand its meaning. In order to have a connection to perform this on, you would have had to have connected either to a single node or a cluster.